packages/astro/src/actions/runtime/virtual/get-action.ts

```diff
@@ -11,7 +11,7 @@ export async function getAction(
 ): Promise<ActionClient<unknown, ActionAccept, ZodType> | undefined> {
 	const pathKeys = path.replace('/_actions/', '').split('.');
 	// @ts-expect-error virtual module
-	let { server: actionLookup } = await import('astro:internal-actions');
+	let { server: actionLookup = {} } = await import('astro:internal-actions');
```

Comment:
Do we have a case where `server` is `undefined`? If so, I think it should be an error instead

Reply:
Ah, I added that from an older prototype, but it's no longer possible to get `undefined`. I can add an internal error to flag when `actionLookup` is unexpectedly undefined or some other type